src/astro/sql/operators/sql_decorator.py

```diff
@@ -204,29 +208,25 @@ def handle_output_table_schema(self, output_table_name, schema=None):
         :return:
         """
         schema = schema or SCHEMA
-        if self.conn_type == "postgres" and self.schema:
+        if self.conn_type in ["postgres", "bigquery"] and self.schema:
```

Comment:
should we use the newly created constants here?

Reply:
Will address this in refactor